synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -985,74 +998,98 @@ CanvasInterface::import_sequence(
 			}
 			
 			String short_filename = CanvasFileNaming::make_short_filename(get_canvas()->get_file_name(), filename);
-			
+
 			try {
 				layer = add_layer_to("Import",get_canvas());
 				int w, h;
 				if (!layer)
 					throw int();
 				if (!layer->set_param("filename", ValueBase(short_filename)))
 					throw int();
+
+
+				curr_layer= modules::lyr_std::Import::Handle::cast_dynamic(layer);
+				cur_surface=cur_lock.get_handle();
+
+				//assuming is_same_as to be the pseudo function to compare two bitmap layers
```

Comment:
this comment is obsolete now, right?

Reply:
I actually removed it :grimacing: I'll make a fresh commit asap